data-jdbc/build.gradle

```diff
@@ -79,7 +79,7 @@ micronaut {
         additionalModules.add(KnownModules.JDBC_POSTGRESQL)
         additionalModules.add(KnownModules.JDBC_MYSQL)
         additionalModules.add(KnownModules.JDBC_MARIADB)
-        additionalModules.add(KnownModules.JDBC_ORACLE_XE)
+        additionalModules.add("jdbc-oracle-free")
```

Comment:
@alvarosanchez Do you know when this field becomes available to the apps using testresources? I tried `KnownModules.JDBC_ORACLE_XE` but micronaut-data (or micronaut-sql) does not see it. Locally, after clearing gradle cache I am still pulling 2.3.3 testresources buildtools (expected 2.4.0) and can't use this field.
For now we can use this hard coded value I guess.

Reply:
It is very strange, the BOM aligns all versions: https://central.sonatype.com/artifact/io.micronaut.testresources/micronaut-test-resources-bom/2.4.0

Reply:
`micronaut-test-resources-build-tools` is integrated by the build plugins. If you update the version of test resources in the build script, it doesn't update the version that the Gradle plugin builds against, so `KnownModules` is still whatever version the Micronaut Gradle plugin was compiled against. Therefore, you'll have to wait for a new Gradle plugin release which integrates with the latest test-resources-build-tools

Reply:
Thanks @melix 
So I guess we will wait for gradle 4.4.0 version that will have 2.4.0 buildtools.